pkg/chains/tron/txm.go

```diff
@@ -4,14 +4,15 @@ import (
 	"fmt"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-evm/pkg/config"
 	"github.com/smartcontractkit/chainlink-evm/pkg/config/toml"
 	"github.com/smartcontractkit/chainlink-evm/pkg/keys"
 	tronkeystore "github.com/smartcontractkit/chainlink-tron/relayer/keystore"
 	tronclient "github.com/smartcontractkit/chainlink-tron/relayer/sdk"
 	trontxm "github.com/smartcontractkit/chainlink-tron/relayer/txm"
 )
 
-func ConstructTxm(logger logger.Logger, nodes []*toml.Node, keystore keys.Store) (*trontxm.TronTxm, error) {
+func ConstructTxm(logger logger.Logger, cfg *config.ChainScoped, nodes []*toml.Node, keystore keys.Store) (*trontxm.TronTxm, error) {
```

Comment:
Let's pass a more specific interface, or even just `limitMultiplier` since that is the only field used currently.

Reply:
I think we usually do this:
```suggestion
type Config interface {
	LimitMultiplier() float64
}

func ConstructTxm(logger logger.Logger, cfg Config, nodes []*toml.Node, keystore keys.Store) (*trontxm.TronTxm, error) {
```
Then the caller can pass `cfg.EVM().GasEstimator()`.

Reply:
I was wondering if we might use any other config in the future. @KodeyThomas  if we don't need anything else I can just pass limitMultiplier. Please confirm?

Reply:
For now I've updated it to just pass the energyMultiplier directly